src/lsstseries/analysis/structurefunction2.py

```diff
@@ -124,188 +135,11 @@ def calc_sf2(
             dts.append(np.hstack(res[0]))
             sf2s.append(np.hstack(res[1]))
 
-    if combine:
+    if argument_container.combine:
         idstack = ["combined"] * len(np.hstack(ids))
     else:
         idstack = np.hstack(ids)
     sf2_df = pd.DataFrame(
         {"lc_id": idstack, "band": np.hstack(bands), "dt": np.hstack(dts), "sf2": np.hstack(sf2s)}
     )
     return sf2_df
-
-
-def _sf2_single(times, fluxes, errors, bins=None, combine=False, method="size", sthresh=100):
```

Comment:
The rest of the functions in this file have been moved to either `BasicStructureFunctionCalculator` or `StructureFunctionCalculator`